idm/idm.go

```diff
@@ -38,7 +38,7 @@ func (i *Idm) GetID() (uint64, error) {
 	if i.handle == nil {
 		return 0, errors.New("ID set is not initialized")
 	}
-	ordinal, err := i.handle.SetAny()
+	ordinal, err := i.handle.SetAny(false)
```

Comment:
don't we need the same sequential allocation for the vxlan? else we are going to hit the vxlan interface `file already exists` correct?